CME/dataobjects/CMEProvider.php

```diff
@@ -67,6 +68,27 @@ public function loadByShortname($shortname)
 	}
 
 	// }}}
+	// {{{ public function getCreditTitle()
+
+	public function getCreditTitle($hours, $credit_count = 1, $is_free = false)
+	{
+		$locale = SwatI18NLocale::get();
+		return sprintf(
+			SwatString::minimizeEntities(
+				CME::_('%s%s %s%s%s certified by %s')
+			),
+			SwatString::minimizeEntities($locale->formatNumber($hours)),
+			$is_free ? ' '.CME::_('Free') : '',
+			'<em>',
+			(abs($hours - 1.0) < 0.01)
+				? SwatString::minimizeEntities($this->credit_title)
+				: SwatString::minimizeEntities($this->credit_title_plural),
+			'</em>',
```

Comment:
I don't think this is an issue. We do it custom for AAFP credits anyway. We could add a boolean property to Provider, or just a `isCreditItalicisized()` method that can be overridden for some credits if necessary though.